spi/src/main/java/org/eclipse/dataspaceconnector/spi/system/ServiceExtension.java

```diff
@@ -23,6 +23,13 @@
  */
 public interface ServiceExtension extends SystemExtension {
 
+    /**
+     * Returns the name of the service, by default is its class name
+     */
+    default String name() {
+        return this.getClass().getSimpleName();
```

Comment:
I think it should be the FQN to avoid name clashes

Reply:
like `serviceExtensionName`? which name clashes could happen?

Reply:
oh, I realized, you talked about the result, not the method name. wouldn't be that too much verbose and confusing on the logs?

Reply:
Per my other comment, for our extensions, I think we should always implement a name and return a user/operator friendly extension name like we currently do. The class name doesn't tell much but makes sense as a fallback default. However, this should be an FQN (package name included) since it is possible for two extension names to clash otherwise. I don't think it will be too verbose.